sdk/monitor/monitor-query/review/monitor-query.api.md

```diff
@@ -101,19 +104,24 @@ export interface LogsQueryOptions extends OperationOptions {
     includeQueryStatistics?: boolean;
     includeVisualization?: boolean;
     serverTimeoutInSeconds?: number;
+    throwOnAnyFailure?: boolean;
 }
 
 // @public
 export interface LogsQueryResult {
     error?: ErrorInfo;
+    logsQueryResultStatus?: LogsQueryResultStatus;
     statistics?: Record<string, unknown>;
     tables: LogsTable[];
     visualization?: Record<string, unknown>;
 }
 
+// @public
+export type LogsQueryResultStatus = "Partial" | "Success" | "Failed";
```

Comment:
So if I understand correctly:

A batch can be successful, partially successful, or failed
A single query can be successful, partially successful, or failed 

Right? If so, under what scenarios does a single query partially succeed?
And, if you rename the batch status union to "Success" | "Partial Success" | "Failure" you can be consistent in both and use the same type...

Reply:
yes for - both the individual queries in a batch and the single queries : 
 - scenarios for partial success - are when we get data in the "table" property and there is some information in the "error"  field by the Kusto query results - https://docs.microsoft.com/en-us/azure/data-explorer/kusto/concepts/partialqueryfailures

The batchResultStatus is specific to JS - to indicate if all the queries that were run in the batch - whether all failed or or succeeded or there were some successes or failures

Reply:
removed the batchResultStatus